### PR TITLE
Add link to blog to `mutation` FAQ entry (Addresses #2332)

### LINKED ISSUE #2332: `mutation` FAQ entry should link to `2022-01-11-cwa-omikron` blog
## Where to find the issue
- https://www.coronawarn.app/de/faq/#mutation
- https://www.coronawarn.app/en/faq/#mutation

## Describe the issue
The FAQ entry mentioned above should also link to the blog post https://www.coronawarn.app/de/blog/2022-01-11-cwa-omikron/ / https://www.coronawarn.app/en/blog/2022-01-11-cwa-omikron/

## Suggested change
Add a new line to the "UPDATE" section of the entry.

## Additional information
I will provide a PR for this.

### PR DESCRIPTION
**Pull request by Ein-Tim**
This PR adds a link to the blog https://www.coronawarn.app/de/blog/2022-01-11-cwa-omikron / https://www.coronawarn.app/en/blog/2022-01-11-cwa-omikron to the FAQ entry https://www.coronawarn.app/de/faq/#mutation / https://www.coronawarn.app/en/faq/#mutation.

<details>
<summary>Preview of the changes</summary>

| German | English |
|---|---|
| DE | EN |

</details>

---

**Closes #2332**

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -1811,7 +1811,8 @@
                         "anchor": "mutation",
                         "active": false,
                         "textblock": [
-                            "<b>UPDATE</b><br/>To date, the risk assessment was improved and adapted on the 23.02.2021, on the 19.03.2021 and on the 16.04.2021. More information can be found in our blog:",
+                            "<b>UPDATE</b><br/>Information on the Corona-Warn-App and omicron can be found in this blog post: <a href='../blog/2022-01-11-cwa-omikron' target='_blank'>Wird die CWA an die Omikron-Variante angepasst?</a> (German only)",
```
Comment (MikeMcC399) [inline src/data/faq.json:1814]: "omicron" => "Omicron"
It is a proper noun and should be capitalized.
Reference: 
https://www.who.int/news/item/26-11-2021-classification-of-omicron-(b.1.1.529)-sars-cov-2-variant-of-concern
"Based on the evidence presented indicative of a detrimental change in COVID-19 epidemiology, the TAG-VE has advised WHO that this variant should be designated as a VOC, and the WHO has designated B.1.1.529 as a VOC, named Omicron."

(German only) would look better with a full stop to end the sentence.

Comment (Ein-Tim) [reply src/data/faq.json:1814]: I will take this into account, thanks.

Comment (Ein-Tim) [inline src/data/faq.json:1814]:
```suggestion
                            "<b>UPDATE</b><br/>Information on the Corona-Warn-App and Omicron can be found in this blog post: <a href='../blog/2022-01-11-cwa-omikron' target='_blank'>Wird die CWA an die Omikron-Variante angepasst?</a> (German only).",
```

**Review by Ein-Tim — COMMENTED** [review]
Review from @MikeMcC399.

File: src/data/faq.json (hunk 1 above)
Comment (Ein-Tim) [reply src/data/faq.json]: Done in https://github.com/corona-warn-app/cwa-website/commit/ac5b1b5a05053a386c3850a3a8bcbe248b6a935a.
Sorry for the default name of the commit, I used a workaround to make this changes as I'm not near my machine (which I provided this PR with) at the moment.

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@Ein-Tim @MikeMcC399 Thanks a lot! Looks good!

**Pull request merged**